packages/react-components/src/index.ts

```diff
@@ -26,6 +26,24 @@ export type {
   FluentProviderSlots,
   FluentProviderState,
 } from '@fluentui/react-provider';
+export {
+  createCustomFocusIndicatorStyle,
```

Comment:
```suggestion
```

Reply:
I can see situations where customers need to create focus indicator styles... however personally I think we can leave it out for now IMO